connection.cfg

```diff
@@ -1,3 +1,5 @@
 accountName <account-name-here>
-accountKey <account-key-here>
+accountKey <account-name-here-delete-next-line>
```

Comment:
account-key-here... typo

Reply:
Deleting one of these is not required, right ?

Reply:
Yes it is.

Reply:
Why can't we change the flow so that we take accountKey if available, then switch to SAS... And still accept both.. Same goes to environment variables too. Accept both methods, but prefer AccountKey

I am sure there will be users who forget to remove one of these lines and end up failing.

Reply:
When I discussed with Adam, he wanted to fail if both were specified. The error message you get if both (or neither) is specified is very clear that exactly one should be present.

Reply:
I definitely prefer to fail rather than to have to guess.  Could we just add a comment in the sample connection.cfg, explaining?  It looks to me like the parser will ignore any lines it doesn't recognize, so we could just add a line, something like:
\# Please provide either an account key or a SAS token, and delete the other line: